### DISCUSSION TITLE
Osu Testflight (iOS)

### DISCUSSION THREAD

**Discussion by ElfieEllie** [ppy/osu#21223] [Ideas] [answered]
Hey, peppy can you please update the link for osu for iOS/ test flight? which is: https://osu.ppy.sh/home/testflight
thanks
let me know when updated pls

**Comment by peppy** [comment, marked as answer]
I currently remove inactive testers from testflight every time a new lazer release is pushed out. You can follow me on twitter for more details. I usually tweet whenever I run a cleanup process.

**Reply by ElfieEllie** [reply]
any estimate of when's next cleanup? 🥺
